src/components/visualizations/chart/highcharts/test/getOptionalStackingConfiguration.spec.ts

```diff
@@ -98,7 +98,42 @@ describe('getOptionalStackingConfiguration', () => {
             });
         });
 
-        it('should return series config with normal stacking for dual axis', () => {
+        it('should return series with stacking config with normal stacking for dual axis', () => {
+            const chartOptions = {
+                type: VisualizationTypes.COLUMN,
+                yAxes: [{}, {}]
+            };
+            const config = {
+                yAxis: [{}, {}],
+                series: [
+                    ...Array(2).fill({ yAxis: 0 }),
+                    ...Array(2).fill({ yAxis: 1 })
+                ]
+            };
+            const chartConfig = { stackMeasures: true };
+
+            const result = getStackMeasuresConfiguration(chartOptions, config, chartConfig);
+            expect(result).toEqual({
+                stackMeasuresToPercent: false,
+                series: [
+                    ...Array(2).fill({
+                        yAxis: 0, // primary Y axis
+                        stack: 0,
+                        stacking: 'normal'
+                    }),
+                    ...Array(2).fill({
+                        yAxis: 1, // secondary Y axis
+                        stack: 1,
+                        stacking: 'normal'
+                    })
+                ],
+                yAxis: Array(2).fill({
+                    stackLabels: { enabled: true }
+                })
+            });
+        });
+
+        it('should return series without "stackMeasures" config with one measure in each axis for dual axis', () => {
             const chartOptions = {
```

Comment:
this `chartOptions` and `config` are used in two tests, why not to define them outside and share them. Try to keep tests as compact as possible

Reply:
I fixed by using `it.each()` for this case